doc/2/api/controllers/document/update-by-query/index.md

```diff
@@ -0,0 +1,131 @@
+---
+code: true
+type: page
+title: updateByQuery
+---
+
+# updateByQuery
+
+Updates documents matching the provided search query. 
+
+Documents updated that way trigger real-time notifications.
+
+## Limitations
+
+The request fails if the number of documents returned by the search query exceeds the `documentsWriteCount` server configuration (see the [Configuring Kuzzle](/core/2/guides/essentials/configuration) guide).
+
+To update a greater number of documents, either change the server configuration, or split the search query.
+
+---
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/<index>/<collection>/_queryUpdate
```

Comment:
After thinking, just a `PUT` request on `/_query` would be better to keep consistency with deleteByQuery 
```suggestion
URL: http://kuzzle:7512/<index>/<collection>/_query
```